documentation/modules/proc-uninstalling.adoc

```diff
@@ -18,7 +20,7 @@ ifeval::["{cmdcli}" == "oc"]
 endif::[]
 
 ifeval::["{cmdcli}" == "oc"]
-. Delete "cluster level" resources:
+. Delete the cluster-level resources:
```

Comment:
@k-wall, should this be "cluster-level custom resources"?

Reply:
@jenmalloy no, it is correct as it stands.